docs/updating/update-to-46.md

```diff
@@ -14,14 +14,72 @@ modified_at: 2025-06-16
 	You may try removing the `package-lock.json` or `yarn.lock` files (if applicable) and reinstalling all packages before rebuilding the editor. For best results, make sure you use the most recent package versions.
 </info-box>
 
-## Update to CKEditor&nbsp;5 v46.0
+## Update to CKEditor&nbsp;5 v46.0.0
 
 Released on xxx, 2025. ([See full release notes](https://github.com/ckeditor/ckeditor5/releases/tag/v46.0.0))
 
 Below are the most important changes that require your attention when upgrading to CKEditor&nbsp;5 v46.0.0.
 
-### First heading
-
 ### Major breaking changes in this release
 
+#### Content area CSS variables renamed to --ck-content-* prefix
```

Comment:
Should we put all of it in this section? I believe we should list it at the top and just mention the BCs here, like, one-line mention changelog style.